intellij/src/saros/intellij/ui/views/buttons/ToolbarButton.java

```diff
@@ -1,17 +1,26 @@
 package saros.intellij.ui.views.buttons;
 
+import com.intellij.ui.JBColor;
 import com.intellij.util.ui.UIUtil;
+import java.awt.Color;
+import javax.swing.BorderFactory;
 import javax.swing.ImageIcon;
 import javax.swing.JButton;
 
 /** Common class for Toolbar button implementations. */
 abstract class ToolbarButton extends JButton {
 
+  // Self-adjusting references to the current IDE color
+  static final Color FOREGROUND_COLOR = JBColor.foreground();
+  static final Color BACKGROUND_COLOR = JBColor.background();
+
   /** Creates a button with the specified actionCommand, Icon and toolTipText. */
   ToolbarButton(String actionCommand, String tooltipText, ImageIcon icon) {
     setActionCommand(actionCommand);
     setButtonIcon(icon);
     setToolTipText(tooltipText);
+
+    setBorder(BorderFactory.createLineBorder(JBColor.border(), 1, true));
```

Comment:
You could also try to override the paintBorder method (doing nothing) or override the setBorder method to do nothing if you do not want any border.

Reply:
Thanks for the hint. But after comparing the two alternatives, having a border is definitely preferable to not having a border. When the buttons are disabled, they have the same color as the IDE frame in some themes, meaning they can't be differentiated from the frame. This makes it hard to understand that they actually are buttons.